src/sql/src/parsers/alter_parser.rs

```diff
@@ -51,6 +51,17 @@ impl<'a> ParserContext<'a> {
                     parser.peek_token()
                 )));
             }
+        } else if parser.parse_keyword(Keyword::RENAME) {
```

Comment:
Could you add a test case in this mod to check that we are able to parse the rename table SQL? There are some examples in [this mod](https://github.com/GreptimeTeam/greptimedb/blob/e07791c5e88ceae2cd8e154d730efcf7b2914f61/src/sql/src/parsers/alter_parser.rs#L74)